File: Sources/SimERP.Data/DBEntities/VendorProduct.cs
using System;
using System.Collections.Generic;

namespace SimERP.Data.DBEntities
{
    public partial class VendorProduct
    {
        public long RowId { get; set; }
        public int ProductId { get; set; }
        public int VendorId { get; set; }
        public decimal? Price { get; set; }
        public string Notes { get; set; }
        public bool? IsActive { get; set; }
    }
}

File: Sources/SimERP/ClientApp/src/app/lists/supplier/model/vendorproduct.ts
export class VendorProduct {
  RowId: number;
  ProductId = -1;
  VendorId = -1;
  Price: number;
  Notes: string;
  IsActive: boolean;

  // Custom properties
  ProductCode: string;
  ProductName: string;
  UnitName: string;
  CountryName: string;
  ProductCategoryName: string;
}
